lib/efi.py

```diff
@@ -128,8 +155,11 @@ def get_secure_boot_guid(variable: str) -> GUID:
 def cert_to_efi_sig_list(cert):
     """Return an ESL from a PEM cert."""
     with open(cert, 'rb') as f:
-        pem = f.read()
-        cert = x509.load_pem_x509_certificate(pem)
+        cert_raw = f.read()
+        try:
+            cert = x509.load_pem_x509_certificate(cert_raw)
+        except ValueError:
+            cert = x509.load_der_x509_certificate(cert_raw)
```

Comment:
I'm not too much familiar with PEM vs DER, and don't see at first glance what this changes does and why.
Wouldn't that be useful to have in a separate commit, where the rationale for this change would be more easily written?

Reply:
I'll add a comment for this.